workflows/nallo.nf

```diff
@@ -513,6 +489,50 @@ workflow NALLO {
                 }
             }
         }
+
+        // Merge SVs and CNVs if we've called both SVs and CNVs
+        if (!params.skip_cnv_calling) {
+            CALL_SVS.out.ch_multisample_vcf
```

Comment:
Would it be possible to rename this channel to follow the same syntax as the CNVs? I.e `CALL_SVS.out.family_vcf`.
Looks neater and would help with readability.